system_tests/logging_.py

```diff
@@ -80,3 +80,17 @@ def test_create_metric(self):
         metric.create()
         self.to_delete.append(metric)
         self.assertTrue(metric.exists())
+
+    def test_list_metrics(self):
+        metric = Config.CLIENT.metric(
+            DEFAULT_METRIC_NAME, DEFAULT_FILTER, DEFAULT_DESCRIPTION)
+        self.assertFalse(metric.exists())
+        before_metrics, _ = Config.CLIENT.list_metrics()
+        before_names = set(metric.name for metric in before_metrics)
```

Comment:
Yup. I originally thought you could use the existing structures like

```python
delta = [metric for metric in after_metrics if metric not in before_names]
```

but since you've got to grab the name, there is no way getting around creating a new container. SGTM.